packages/react-components/react-breadcrumb-preview/stories/BreadcrumbButton/BreadcrumbButtonAppearanceSubtle.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import { Breadcrumb, BreadcrumbItem, BreadcrumbButton, BreadcrumbDivider } from '@fluentui/react-breadcrumb-preview';
+import { bundleIcon, CalendarMonth20Filled, CalendarMonth20Regular, GridDots20Regular } from '@fluentui/react-icons';
+import { ButtonProps } from '@fluentui/react-components';
+
+const CalendarMonth = bundleIcon(CalendarMonth20Filled, CalendarMonth20Regular);
+
+type Item = {
+  key: number;
+  item?: string;
+  buttonProps?: {
+    'aria-label'?: string;
+    onClick?: () => void;
+    icon?: ButtonProps['icon'];
+    disabled?: boolean;
+  };
+};
+
+const buttonItems: Item[] = [
+  {
+    key: 0,
+    item: 'Button',
+    buttonProps: {
+      onClick: () => window.open('https://react.fluentui.dev/?path=/docs/components-button-button--default', '_blank'),
+    },
+  },
+  {
+    key: 1,
+    item: 'Breadcrumb',
+    buttonProps: {
+      icon: <CalendarMonth />,
+      'aria-label': 'Item 1',
+      disabled: true,
+    },
+  },
+  {
+    key: 2,
+    item: 'Card',
+    buttonProps: {
+      icon: <GridDots20Regular />,
+      onClick: () => window.open('https://react.fluentui.dev/?path=/docs/components-card-card--default', '_blank'),
+    },
+  },
+  {
+    key: 3,
+    item: 'Dropdown',
+    buttonProps: {
+      onClick: () => window.open('https://react.fluentui.dev/?path=/docs/components-dropdown--default', '_blank'),
```

Comment:
This documentation example is about button appearance, do you really need the extra code to handle navigation?

Reply:
You're right. I'll remove it